rt_utils/rtstruct.py

```diff
@@ -25,6 +25,18 @@ def set_series_description(self, description: str):
         """
 
         self.ds.SeriesDescription = description
+        
+    def roi_number(self):        
+         ROI_list = self.ds.StructureSetROISequence
+         ROI_number = []
+         for i in range(len(ROI_list)):
+                ROI_number.append(ROI_list[i].ROINumber.numerator)
```

Comment:
We can use list comprehension here to simplify
```suggestion
        roi_numbers = [roi.ROINumber.numerator for roi in self.ds.StructureSetROISequence ]
```

note we have been using lowercase for ROI, we should remain consistent